src/libsyntax/tokenstream.rs

```diff
@@ -820,98 +929,108 @@ impl Index<usize> for TokenStream {
     type Output = TokenTree;
 
     fn index(&self, index: usize) -> &TokenTree {
```

Comment:
I meant the `index` function, but I was confused with the slice function, this is in a trait, so doesn't have to be private.

So, what potential misuse do you see?